src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import Todo from '../../types/Todo';
+import users from '../../api/users';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const currentUser = users.find(user => user.id === todo.userId);
+
+  return (
```

Comment:
[checklist 5](https://github.com/mate-academy/react_add-todo-form/blob/master/newChecklist.md)
There is no need to download users from server twice in one component. Better prepare all the information about todos in one component and send it to al the others as parameters. This correction is only for visualisation of my thoughts. You can create other ideas of how to send it.
```suggestion
import users from '../../api/users';

type Props = {
  todo: Todo,
  currentUser: User,
};

export const TodoInfo: React.FC<Props> = ({ todo, currentUser }) => {

  return (
```